LayoutTests/ipc/invalid-message-to-web-process-crash.html

```diff
@@ -0,0 +1,19 @@
+<!-- webkit-test-runner [ IPCTestingAPIEnabled=true ] -->
+<p>This test passes if WebKit does not crash.</p>
+<script>
+  if (window.testRunner) {
+    testRunner.dumpAsText();
+    testRunner.waitUntilDone();
+    setTimeout(() => {
+        testRunner.notifyDone();
+    }, 5000);
```

Comment:
I don't think we should have a test that runs for 5 seconds. That's crazy. Can we make this a lot shorter?

Reply:
10ms should be long enough. Or better yet, delete calls to `testRunner.waitUntilDone` and `testRunner.notifyDone` altogether?